### ISSUE TITLE
how to: configure mtar so that after deployment specific operations done to start service

### ISSUE THREAD

**Issue by lucatoldo** [cloudfoundry-incubator/multiapps-cli-plugin#73]
### Description
I'be built an mtar using **mbt** and have tried to deploy it with **cf deploy**.
While it is a java application, is not a springboot and it has many dependencies. Therefore I've built a zip . For the deployment, that zip has to be unzipped and then a specific commandline has to be executed. I do not know where to specify this behavor.

### Your environment
*	MBT version 0.2.10
*	MultiApps CF CLI Plugin version - 2.1.1
*	which CF vendor is used - cf version 6.39.1+efdd631d2.2018-09-18

**Comment by valentinEmpy** [comment]
Hello @lucatoldo ,

Apologies for the inactivity in github issues. 
As the issue is quite old, I have to ask - is your query still relevant?
If I understood correctly - you have difficulties using the multiapps plugin in order to deploy the generater .mtar?

Please take a look at the [Usage section](https://github.com/cloudfoundry-incubator/multiapps-cli-plugin#usage). 

You can get more information about the usage of commands via '`cf [command] --help`' (if you'd like the deploy command: '`cf deploy --help`').

In short, you do not need to unzip the .mtar in order to deploy it.
You can deploy it directly, as also described in the usage section:
`cf deploy ./spring-music.mtar`

As the issue is quite old - I will proceed and close it.
If you need further assistance - please re-open or create a new one.

Regards,
Valentin
